### ISSUE TITLE
Links do not have a discernible name

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#9630]

**Comment by veronicamar** [comment]
From PageSpeeds:
Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users.

<img width="901" alt="Screenshot 2024-07-27 at 2 47 01 pm" src="https://github.com/user-attachments/assets/40f392da-e67f-4ab5-9559-648ed8abd0be">

Ensure that all link names are accessible. It may be possible that the inner link text is not visible to a screen reader, that there are duplicate link labels, or that the link is not focusable.

Ensure all links can receive programmatic focus; for example, avoid device-specific events (for example, onmouseover).

To ensure all link text is visible to screen readers, link text cannot be hidden (e.g. with display: none or aria-hidden="true").

To ensure all links can receive programmatic focus, avoid device-specific Javascript events such as onmouseover(), mouseover(), hover(), onmouseout(), mouseout(). Replace these with device-independent events such as onfocus(), focus(), onblur(), or blur().

Do not modify the style of the links to suppress the change in style when a link is the object of programmatic focus. Modifying link styles removes the capability for sighted keyboard users to know where they are on the page. Furthermore, ensure you are creating real links using the a element with the href attribute.

**Comment by srivastavaarpit977** [comment]
Hey @cdrini I would like to work on this issue....Plz assign this issue to me.!!

**Comment by Jash2606** [comment]
Hey @scottbarnes @cdrini  ,  I have updated the lang attribute as mentioned in issue.
Please review the changes and let me know if any further updates are needed.
Thank you!

**Comment by rebecca-shoptaw** [comment]
@veronicamar Thanks for the info!! I'll do some digging re: where this is showing up and how best to approach the problem, then update the issue and assign someone.
